test/x/exercise_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../test_helper'
+require_relative '../x_helper'
+
+module X
+  class ExerciseTest < Minitest::Test
+    def test_test_files
+      f= './test/fixtures/xapi_v3_exercise_tests.json'
+      Xapi.stub(:get, [200, File.read(f)]) do
+        expected = Exercise::TestFiles.find('ruby', 'saddle-points')
+
+        assert_equal "Animal", expected.language
+        assert_equal "Dog", expected.name
+        assert_equal "This is dog.", expected.blurb
+        assert_kind_of Hash, expected.files
+        assert_equal ["a_dog.animal", "a_dog_2.animal"], expected.files.keys
+      end
+    end
+
+    def test_readme
+      f = './test/fixtures/xapi_v3_exercise_readme.json'
+      Xapi.stub(:get, [200, File.read(f)]) do
+        expected = Exercise::Readme.find('ruby', 'saddle-points')
```

Comment:
Again, I'm not sure why this is looking for ruby/saddle-points and getting fake/three.